v1.1-alpha.20170601.md

```diff
@@ -0,0 +1,72 @@
+---
+title: What&#39;s New in v1.1-alpha.20170601
+toc: false
+summary: Additions and changes in CockroachDB version v1.1-alpha.20170601
+---
+
+## Jun 1, 2017
+
+Get future release notes emailed to you:
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1-alpha.20170601.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1-alpha.20170601.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1-alpha.20170601.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1-alpha.20170601.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### General Changes
+
+- CockroachDB now uses RocksDB 5.3.4 ([#15696](https://github.com/cockroachdb/cockroach/pull/15696)), protobuf 3.3.0 ([#15522](https://github.com/cockroachdb/cockroach/pull/15522)), and a newer version of gRPC ([#15991](https://github.com/cockroachdb/cockroach/pull/15991))
+- Non-release binaries no longer send crash reports. [#15823](https://github.com/cockroachdb/cockroach/pull/15823)
+- Some `make` targets have changed to follow convention. `make check` is now called `make lint`.
+- The Kubernetes templates now use the latest production release instead of tracking testing releases. [#15878](https://github.com/cockroachdb/cockroach/pull/15878)
+
+### SQL Language Changes
+
+- The `information_schema.user_privileges` table is now supported. [#15745](https://github.com/cockroachdb/cockroach/pull/15745)
+- The `ON UPDATE RESTRICT` and `ON DELETE RESTRICT` modifiers can now be used when creating foreign keys (this behavior is the default, but now it can be specified explicitly). [#15815](https://github.com/cockroachdb/cockroach/pull/15815)
+- The hash functions `sha512()`, `fnv32()`, `fnv32a()`, `fnv64()`, `fnv64a()`, `crc32ieee()`, and `crc32c()` are now supported. All hash functions now accept multiple arguments and arguments of type `BYTES`. [#15828](https://github.com/cockroachdb/cockroach/pull/15828) [#15859](https://github.com/cockroachdb/cockroach/pull/15859)
+- Casts to type `DECIMAL` can now specify precision and scale. [#15834](https://github.com/cockroachdb/cockroach/pull/15834)
```

Comment:
```Casts to type `DECIMAL` now respect precision and scale``` may be more accurate, since they were ignored before. This could also go under bugs, unsure.